apps/client/src/plugins/Coordinates/CoordinatesTransformRow.js

```diff
@@ -67,6 +81,110 @@ class CoordinatesTransformRow extends React.PureComponent {
     });
   }
 
+  getCoordinates(coordinateFormatTitle) {
+    let inputX, inputY;
+    if (coordinateFormatTitle === "SWEREF 99 12 00") {
+      inputX = document.getElementsByName("numberformatX")[0].value;
+      inputY = document.getElementsByName("numberformatY")[0].value;
+    } else if (coordinateFormatTitle === "SWEREF 99 TM") {
+      inputX = document.getElementsByName("numberformatX")[1].value;
+      inputY = document.getElementsByName("numberformatY")[1].value;
+    } else {
+      inputX = document.getElementsByName("numberformatX")[2].value;
+      inputY = document.getElementsByName("numberformatY")[2].value;
+    }
+
+    inputX = inputX.replace(/\s/g, "");
+    inputY = inputY.replace(/\s/g, "");
+
+    return { inputX, inputY };
+  }
+
+  handleCopyToClipBoard(coordinateFormatTitle) {
+    //We get the correct input fields values using the title of the coordinate system
+    const { inputX, inputY } = this.getCoordinates(coordinateFormatTitle);
+
+    // We check if the values have any numbers, and if not exit the function early...
+    // and give an alert to the user
+    if (inputX === "" || inputY === "") {
+      // Display a message if any of the fields are empty
+      this.props.enqueueSnackbar("Kopiering misslyckades, fälten är tomma", {
+        variant: "error",
+      });
+      return;
+    }
+
+    // Set the string to be copied from the two X and Y values
+    const coordinatesString = `${inputX},${inputY}`;
+
+    // We create a temporary element to store the and copy the coordinateString
+    const input = document.createElement("input");
+    input.value = coordinatesString;
+
+    // Make the element non-interactive and hide it
+    input.setAttribute("readonly", "");
+    input.style.position = "absolute";
+    input.style.left = "-9999px";
+
+    // We copy the string from the element, alert the user of successful copying...
+    document.body.appendChild(input);
+    input.select();
+    document.execCommand("copy") &&
+      this.props.enqueueSnackbar("Kopiering till urklipp lyckades!", {
```

Comment:
Maybe:
"Kopierat koordinaten till urklipp"
Opinions?

Reply:
Changed this too. It now says "Koordinaten kopierades till urklipp"